server/api/static.js

```diff
@@ -0,0 +1,41 @@
+import { readFile } from 'fs'
+import path from 'path'
+import { URL } from 'node:url'
+
+const CLIENT_DIR = '../../client'
+const DIST_DIR = '/dist'
+const ASSETS_DIR = '/assets'
+
+const MIME_TYPES = {
+  '.js': 'text/javascript',
+  '.css': 'text/css',
+  '.json': 'application/json',
+  '.png': 'image/png',
+  '.jpg': 'image/jpg',
+  '.svg': 'image/svg+xml'
+}
+
+export default async function handleStatic(req, res) {
+  let filePath
+
+  if (req.url.includes(ASSETS_DIR)) {
+    filePath = new URL(`${CLIENT_DIR}${req.url}`, import.meta.url)
+  } else {
+    filePath = new URL(`${CLIENT_DIR}${DIST_DIR}${req.url}`, import.meta.url)
+  }
+
+  let extname = path.extname(req.url)
+
+  readFile(filePath, (error, content) => {
+    if (!error) {
```

Comment:
1. We should check allow-list of files extensions and paths (and do not try to read them)
2. It is not a good practice to try to unknown read file. It is better to check `fs.exists` and then return `404` and return `500` on reading error.

Reply:
Thanks!

I used `fs.access`, `fs.exists` is deprecated